digitalocean/Image.py

```diff
@@ -62,3 +62,6 @@ def rename(self, new_name):
 
     def __str__(self):
         return "%s %s %s" % (self.id, self.name, self.distribution)
+
+    def __repr__(self):
+        return "%s %s %s" % (self.id, self.distribution, self.name)
```

Comment:
I wonder if something like this might be better:

    return "< %s %s %s >" % (self.id, self.distribution, self.name)

This would be much more clear to me that this is an object and not a list of strings.